<ui version="4.0" >
 <author>Robert Gruber &lt;rgruber@users.salomon.at></author>
 <class>SnippetViewBase</class>
 <widget class="QWidget" name="SnippetViewBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>377</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="KLineEdit" name="filterText" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Define filter here</string>
       </property>
       <property name="frame" >
        <bool>false</bool>
       </property>
       <property name="showClearButton" stdset="0" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbFilter" >
       <property name="toolTip" >
        <string>Apply filter</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="snippetTree" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterText</tabstop>
  <tabstop>tbFilter</tabstop>
  <tabstop>snippetTree</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>filterText</sender>
   <signal>returnPressed()</signal>
   <receiver>tbFilter</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>132</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>185</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
